internal/worker/externalcontrollerupdater/export_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package externalcontrollerupdater
+
+import (
+	"github.com/juju/worker/v3"
+	gc "gopkg.in/check.v1"
+)
+
+func GetWorkerRunner(c *gc.C, w worker.Worker) *worker.Runner {
```

Comment:
We're trying to avoid using export_test
If we need access to internals, put the test in the package not package_test.
Or pass in a callback or some other way of observing the behaviour.